Tasks/Common/webdeployment-common-v2/packageUtility.ts

```diff
@@ -29,12 +29,36 @@ export class Package {
     constructor(packagePath: string) {
         this._path = PackageUtility.getPackagePath(packagePath);
         this._isMSBuildPackage = undefined;
+        let workingDirectory = tl.getVariable("system.defaultworkingdirectory");
+        try {
+            if (workingDirectory && this._path.indexOf(workingDirectory) == 0) {
+                let relativePackagePath = packagePath.substring(workingDirectory.length);
```

Comment:
Should this be this._path instead of packagePath. I assume packagePath can contain patterns like * etc

Reply:
yes it should be this._path thats what i have used in if() condition but ended up using packagePath here. Will change it,